app/graphql/mutations/delete_listing.rb

```diff
@@ -0,0 +1,23 @@
+module Mutations 
+  class DeleteListing < Mutations::BaseMutation
+    description "Delete an existing listing" 
+
+    argument :id, Integer, required: true 
+
+    field :listing, Types::ListingType, null: true 
+    field :error, [String], null: false 
+
+    def resolve(args)
+      error = []
+      begin
+        deleted_listing = Listing.destroy(args[:id])
+      rescue 
```

Comment:
I'm curious what the begin and rescue are doing.

Reply:
Seconded!